programs/autocrat_v0/src/lib.rs

```diff
@@ -145,10 +150,13 @@ pub mod autocrat_v0 {
             AutocratError::InvalidMarket
         );
 
-        // this should also be checked by `openbook_twap`, but why not take the
-        // precaution?
+        let current_time = Clock::get().unwrap().unix_timestamp as i64;
+
+        // The market expires a minimum of 7 days after the end of the proposal.
```

Comment:
7 or 10?

Reply:
3 days of proposal + 7 days after the end of the proposal

10 days in total minimum.